app/src/main/java/com/example/oddhours/ui/shifts/ChildAdapter.kt

```diff
@@ -134,10 +133,11 @@ class ChildAdapter(private var shiftsList: List<ShiftsModel>, val context: Conte
                         dayOfYear = 0
                         val helper = Helper()
                         dayOfYear = helper.calculateDayOfTheYear(monthOfYear, dayOfMonth, year)
-                        // DEBUG
-                        // Log.d(TAG, "startDate: year: $year, month: ${monthOfYear}, day: $dayOfMonth")
-                        // Log.d(TAG, "startDate after setting via the button: ${startDate.time}")
                     }, year, month, day)
+                    // DEBUG
```

Comment:
i may have forgotten but i believe these values are different outside of the datePickerDialog so maybe we should have the debug logging back inside the dialog

Reply:
@oyiza could you explain what you mean by logging back inside the dialog. like move the log lines after line 135?

Reply:
yeah exactly